.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: Release
+
+on:
+  release:
+    types: [published]
```

Comment:
maybe we should add a manual trigger :thinking:

Reply:
If we go down the commit route instead of using release with versions, we should do that, yes